rpcs3/Emu/RSX/D3D12/D3D12Formats.cpp

```diff
@@ -110,7 +110,7 @@ D3D12_STENCIL_OP get_stencil_op(u32 op) noexcept
 	case CELL_GCM_INCR: return D3D12_STENCIL_OP_INCR;
 	case CELL_GCM_DECR: return D3D12_STENCIL_OP_DECR;
 	case CELL_GCM_INVERT: return D3D12_STENCIL_OP_INVERT;
-	case CELL_GCM_INCR_WRAP:
+	case CELL_GCM_INCR_WRAP: return D3D12_STENCIL_OP_INCR;
```

Comment:
@Zangetsu38 Please at least add a comment about CELL_GCM_INCR_WRAP being used in Shovel Knight.